.github/workflows/run-tests.yml

```diff
@@ -442,6 +442,14 @@ jobs:
             qt-version: 5
 
     steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
```

Comment:
This is selected using the tag in other places too, but don't blame me if it ever gets compromised and the QGIS secrets get stolen.

Reply:
Personally I hope no QGIS secrets are ever put in the github.com (microsoft) hands

Reply:
These are: https://github.com/qgis/QGIS/blob/master/.github/workflows/build-docker.yml#L46-L48.